Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -812,9 +812,9 @@ public struct ObjectID: BSONValue, Equatable, CustomStringConvertible, Codable {
 
     /// Initializes an `ObjectID` from the provided hex `String`. Returns `nil` if the string is not a valid ObjectID.
```

Comment:
we need to update this docstring to indicate that we throw rather than returning nil.